website/src/pages/code-search-guide.tsx

```diff
@@ -0,0 +1,189 @@
+                        
+                    <p> Our intention with this guide is to help every dev and development team navigate their codebase with greater ease so you can spend more time creating and less time agonizing over unintended rabbit holes. Let's get started! </p>
+
+                    <div className="container my-4 video-embed embed-responsive embed-responsive-16by9 ">
+                        <iframe
+                            class="embed-responsive-item"
+                            src="https://www.youtube-nocookie.com/embed/J9k7l5W1qbk?autoplay=0&amp;cc_load_policy=0&amp;start=0&amp;end=0&amp;loop=0&amp;controls=1&amp;modestbranding=1&amp;rel=0"
+                            allowfullscreen=""
+                            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                            frameborder="0"
+                            title="Universal code search using regex with Sourcegraph"
+                        ></iframe>
+                    </div>
+
+		            <hr className="my-md-6" id="" />
+            	</div>
+
+            	<div className="col-lg-7 offset-lg-4" id="2">
+                    <h2 className="display-3">2 What is code search?</h2>
+                    <p>
+                        Coming soon.
+                    </p>
+
+                    <div className="container my-4 video-embed embed-responsive embed-responsive-16by9 ">
+                        <iframe
+                            class="embed-responsive-item"
+                            src="https://www.youtube-nocookie.com/embed/J9k7l5W1qbk?autoplay=0&amp;cc_load_policy=0&amp;start=0&amp;end=0&amp;loop=0&amp;controls=1&amp;modestbranding=1&amp;rel=0"
+                            allowfullscreen=""
+                            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                            frameborder="0"
+                            title="Universal code search using regex with Sourcegraph"
+                        ></iframe>
+                    </div>
+
+		            <hr className="my-md-6" id="" />
+            	</div>
+
+            	<div className="col-lg-7 offset-lg-4" id="3">
+                    <h2 className="display-3">3 Why code search?</h2>
+                    <p> Once developers have code search they wonder how they ever lived without it. Being able to quickly find, understand, and fix the code you’re looking for just makes sense when 75% of a developer’s time is spent reading and understanding code. Not surprisingly, this sentiment is quite common and motivating among people who use and build code search tools. Han-Wen Nienhuys, the creator of the Zoekt code search engine, says:</p>
+                        
+                    <blockquote>
```

Comment:
@mgyorffy here is the quote I need styled

Reply:
@content-hacker added quote styling :)